src/server.js

```
@@ -0,0 +1,315 @@
+  });
+
+  server.setRequestHandler(CallToolRequestSchema, async (request, context) => {
+    const { name, arguments: args } = request.params;
+    logger.info({ event: "call_tool", tool_name: name, args }); // Use logger.info, added tool_name for clarity
+
+    try {
+      let result;
+
+      switch (name) {
+        case "create-react-app":
+          const createArgs = CreateReactAppSchema.parse(args);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)